charts/cloudzero-agent/Chart.yaml

```diff
@@ -12,7 +12,8 @@ dependencies:
     version: "5.15.*"
     repository: https://prometheus-community.github.io/helm-charts
     condition: kube-state-metrics.enabled
-  - name: prometheus-node-exporter
-    version: "4.24.*"
-    repository: https://prometheus-community.github.io/helm-charts
-    condition: prometheus-node-exporter.enabled
+  - name: cloudzero-insights-controller
+    version: "0.0.1"
+    repository: https://cloudzero.github.io/cloudzero-charts
```

Comment:
Should this use the `beta` channel for now? Not sure if we need to add the `--devel` flag somehow